src/ocaml_protoc_plugin/service.ml

```diff
@@ -4,12 +4,28 @@ module type Message = sig
   val to_proto: t -> Writer.t
 end
 
+module type Rpc = sig
+  module Request : Message
+  module Response : Message
+  val name' : unit -> string
+end
+
 let make_client_functions (type req) (type rep)
     ((module Request : Message with type t = req),
-     (module Reply : Message with type t = rep)) =
-  Request.to_proto, Reply.from_proto
+     (module Response : Message with type t = rep)) =
+  Request.to_proto, Response.from_proto
+
+let make_client_functions' (type req) (type rep)
+    (module R : Rpc with type Request.t = req and type Response.t = rep) =
+  make_client_functions ((module R.Request), (module R.Response))
```

Comment:
```suggestion
```
I don't think this function is needed right now. In the future we could consider adding a functor which takes a `Rpc` module as argument and exposes helper functions